conf/modules.config

```diff
@@ -18,16 +18,65 @@ process {
         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     ]
 
-    withName: SAMPLESHEET_CHECK {
+    withName: BUILD_INTERVALS {
         publishDir = [
-            path: { "${params.outdir}/pipeline_info" },
+            enabled = false
+        ]
+    }
+
+    withName: 'CREATE_INTERVALS_BED' {
+        publishDir       = [
+            enabled: false
+        ]
+    }
+
+    withName: GATK4_MUTECT2 {
+        ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}" : "${meta.id}.${intervals.simpleName}" }
+        ext.args = '-max-mnp-distance 0'
+        publishDir       = [
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/" },
+            pattern: "*{vcf.gz,vcf.gz.tbi}",
+            saveAs: { meta.num_intervals > 1 ? null : "/mutect2/${meta.id}" }
+        ]
+    }
+
+    withName: 'GATK4_MERGEVCFS' {
+        publishDir       = [
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/mutect2/${meta.id}" },
+            pattern: "*{vcf.gz,vcf.gz.tbi}"
+        ]
+    }
+
+    withName: GATK4_GENOMICSDBIMPORT {
+        publishDir = [
+            path: { "${params.outdir}/genomicsdbimport/${meta.id}/" },
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
 
-    withName: FASTQC {
-        ext.args = '--quiet'
+    withName: GATK4_CREATESOMATICPANELOFNORMALS {
+        ext.prefix = "panel_of_normals"
+        publishDir = [
+            path: { "${params.outdir}/mutect2_panelofnormals/" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: CNVKIT_BATCH {
+        ext.args = [ "",
+                params.assume_male ? "-y" : "", // if male donors, then X chromosome will have +1 log-CNR, otherwise male would have -1 chrX (see CNCkit documentation)
```

Comment:
```suggestion
                params.assume_male ? "-y" : "", // if male donors, then X chromosome will have +1 log-CNR, otherwise male would have -1 chrX (see CNVkit documentation)
```